packages/talos_cohort_replicator/src/services/statemap_queue_service.rs

```diff
@@ -291,14 +292,21 @@ where
                     StatemapInstallState::Awaiting
                 };
 
+                // Record event when the item was received in queue service
+                event_timings.record_event_timestamp(
+                    ReplicatorCandidateEvent::QueueStatemapReceived,
+                    OffsetDateTime::now_utc().unix_timestamp_nanos(),
+                );
+
                 self.statemap_queue.insert_queue_item(
                     &version,
                     StatemapInstallerHashmap {
-                        timestamp: OffsetDateTime::now_utc().unix_timestamp_nanos(),
+                        // timestamp: OffsetDateTime::now_utc().unix_timestamp_nanos(),
```

Comment:
I can see this is now replaced by `StatemapEvents`, suggest to remove `timestamp`